client/src/settings/db.ts

```diff
@@ -114,18 +75,16 @@ export class MDNOfflineDB extends Dexie {
       whoami:
         "++, username, is_authenticated, email, avatar_url, is_subscriber, settings",
     });
+    this.version(4).stores({
+      collections: null,
+      watched: null,
+      notifications: null,
+    });
```

Comment:
Do we need to deploy this change separately, to avoid that the client removes these stores, while an older version of the PWA still accesses them?

Reply:
Indeed we first should stop using them in the sw and then drop them.